### ISSUE TITLE
MasterBehavior="Split" is not working after updating to XF 4.2 and iPadOS

### ISSUE THREAD

**Issue by mariomoramontero** [xamarin/Xamarin.Forms#7710]
Hi
After upgrading XF to the latest v4.2 and testing our app on the simulator with iPadOs, we noticed that the "split" MasterBehavior has stopped working on our MasterDetailPage (portrait mode). 
This happens on portrait mode with both "Split" and "SplitOnPortrait", but when we turn the device to landscape it works as expected. The Popover mode also works on portrait. 

Also I tried with "IsPresented=true" but doesn't work either.

### Steps to Reproduce

1. Having a MasterDetailPage, Set the property MasterBehavior="Split" or MasterBehavior="SplitOnPortrait"
2. Execute the app on iOS and put device on portrait

### Expected Behavior
Both masterpage and detail must be visible at a time 
### Actual Behavior
Detail is opened fullsize, master is hidden, no hamburger button. When opened by swipe the master is as a popover over the detail.

### Basic Information

- Version with issue: Forms 4.2.0815419 and 4.3.0819712-pre2
- Last known good version: Forms 3.6.0
- IDE: VS 2019 16.3.1
- Platform Target Frameworks: 
  - iOS:  13.2.0.42

- Affected Devices: iPad Air with iOS 13.0 (iPadOS)

**Comment by samhouts** [comment]
@mariomoramontero Can you confirm that this was working on 4.1 using iOS 13? Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by PureWeen** [comment]
duplicate https://github.com/xamarin/Xamarin.Forms/issues/7556

**Comment by breyed** [comment]
A regression test suite run during the beta for each major iOS release to compatibility problems before customers upgrade would save Xamarin developers countless hours. In lieu of that, this workaround code might save at least a bit of time:

``` c#
[assembly: ExportRenderer(typeof(MasterDetailPage), typeof(FixupTabletMasterDetailRenderer))]
class FixupTabletMasterDetailRenderer : TabletMasterDetailRenderer
{
	protected override void OnElementChanged(VisualElementChangedEventArgs e)
	{
		base.OnElementChanged(e);
		PreferredDisplayMode = UISplitViewControllerDisplayMode.AllVisible;
	}
}
```

**Comment by breyed** [comment]
The above workaround is problematic, as described in the comment:
> Restores the master page when MasterDetailPage reappears after popping a modal page. It initially doesn't display and then pops in after a split second. Not the best experience, but passable in the short term.

This is with XF 4.3.0.851321-pre3 on iPadOS 13.1.2. What is the proper way to fix this so that there isn't any jankiness?

**Comment by breyed** [comment]
The bug is fixed in XF 4.3.0.908675. The above workaround is obsolete.
